sound/soc/sof/ipc4-loader.c

```diff
@@ -88,7 +110,11 @@ static size_t sof_ipc4_fw_parse_ext_man(struct snd_sof_dev *sdev)
 				(fm_entry + fw_header->num_module_entries);
 	remaining -= (fw_header->num_module_entries * sizeof(*fm_entry));
 	for (i = 0; i < fw_header->num_module_entries; i++) {
-		memcpy(&fw_module->man4_module_entry, fm_entry, sizeof(*fm_entry));
+		/* ignore modules with null UUID */
+		if (guid_is_null(&fm_entry->uuid)) {
+			fm_entry++;
+			continue;
```

Comment:
can this happen? Isn't this a fundamental error that should result in aborting the code loading?

Reply:
no I actually added this based on what I saw. The wov module has 2 modules , 1 that has the actually UUID for the wov module and another one with null UUID. So this check is intentional here.